API/hermes/hermes.h

```diff
@@ -121,10 +124,12 @@ class HermesRuntime : public jsi::Runtime {
   /// it can be written into the trace for later replay.
   const ::hermes::vm::MockedEnvironment &getMockedEnvironment() const;
 
+#ifdef HERMESVM_API_TRACE
   /// Get a structure representing the execution history (currently just of
   /// GC, but will be generalized as necessary), to aid in debugging
   /// non-deterministic execution.
```

Comment:
I wasn’t entirely sure if this comment is referring to aiding debugging _of_ tracing, in which case this may need to be `HERMESVM_API_TRACE_DEBUG`? For now I went with `HERMESVM_API_TRACE` because e.g. `SynthTrace.h` guards the inclusion of the `hermes/VM/GCExecTrace.h` header using that macro.